int8 offset
int8 yaw_change
int8 depth_change
int8 speed
